docs/recipes/WritingTests.md

```diff
@@ -8,12 +8,39 @@ hide_title: true
 
 # Writing Tests
 
-Because most of the Redux code you write are functions, and many of them are pure, they are easy to test without mocking.
+Because most of the Redux code you write are functions, and many of them are pure, they are easy to test without mocking. However, you should consider whether each piece of your Redux code needs it's own dedicated tests.
+
+:::tip
+
+See [Blogged Answers: The Evolution of Redux Testing Approaches](https://blog.isquaredsoftware.com/2021/06/the-evolution-of-redux-testing-approaches/) for Mark Erikson's thoughts on how Redux testing has evolved from 'isolation' to 'integration' over time.
+
+:::
+
+### Guiding Principles
+
+The guiding principles for testing Redux logic closely follow that of React Testing Library:
+
+> [The more your tests resemble the way your software is used, the more confidence they can give you.](https://twitter.com/kentcdodds/status/977018512689455106) - Kent C. Dodds
+
+In the majority of scenarios, the end-user does not know, and does not care whether Redux is used within the application at all.
+As such, the Redux code can be treated as an implementation detail of the app, without requiring explicit tests for the Redux code in many circumstances.
+
+The general advice for testing an app using Redux is as follows:
+
+- Use integration tests for everything working together. I.e. for a React app using Redux, render a `<Provider>` with a real store instance wrapping the component/s being tested. Interactions with the page being tested should use real Redux logic, with API calls mocked out so app code doesn't have to change, and assert that the UI is updated appropriately.
+- Use basic unit tests _where fitting_ for pure functions that deserve it. I.e. particularly complex reducers or selectors. However, in many cases, these are just implementation details that are covered by integration tests instead.
+
+:::tip
+
+See [Testing Implementation Details](https://kentcdodds.com/blog/testing-implementation-details) for Kent C. Dodds' thoughts on why he recommends avoiding testing implementation details.
+
+:::
 
 ### Setting Up
 
 We recommend [Jest](https://facebook.github.io/jest/) as the testing engine.
```

Comment:
:hand: Did we already have the Jest recommendation here?

I'd like to reword this section to emphasize that Redux can be tested with any test runner, but the examples will use Jest as that's common.

Reply:
Yeah, that bit was existing (I just added a line about `jsdom`). I re-worded it more in line with your suggestion